PluginClasses/Zee5PluggablePlayer.swift

```diff
@@ -42,8 +43,13 @@ public class Zee5PluggablePlayer: APPlugablePlayerBase, ZPAdapterProtocol {
             return nil
         }
         
-        Zee5DownloadManager.shared.initializeDownloadManger()
+
+        print("pluggablePlayerRemoveInline::Videos : \(videos)")
```

Comment:
please remove logs and comment

Reply:
Logs and Comment ? or that download manager initializer?